app/src/main/java/com/nervousfish/nervousfish/activities/SettingsActivity.java

```diff
@@ -130,15 +138,16 @@ private void updateListPreference(final Preference preference, final String stri
      * immediately updated upon calling this method. The exact display format is
      * dependent on the type of preference.
      *
-     * @see #sBindPreferenceSummaryToValueListener
+     * @see #BIND_PREFERENCE_SUMMARY_TO_VALUE_LISTENER
```

Comment:
If you press Ctrl-Q, Intellij shows you the javadoc of the method. The @see is omitted if you wrap it in braces and we do not want that of course